src/style/result.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Chakra+Petch:wght@300&family=Neucha&display=swap');
+
+.result{
+    display: flex;
+    justify-content: center;
+    height: 100vh;
+    font-family: 'Chakra Petch', sans-serif;
```

Comment:
i made some variables in styles.css to use for colors, font families and font sizes. We should use those